docs/themes-list/barbuk.rst

```diff
@@ -92,6 +93,29 @@ Please refer to the following documentation for more information:
    EOF
    chmod 400 /etc/sudoers.d/keepenv
 
+Command duration
```

Comment:
can you please put this under https://github.com/Bash-it/bash-it/blob/master/docs/themes.rst#theme-switches--variables instead? makes more sense as it is a generic feature. You can link from your theme to there as well

Reply:
Done in https://github.com/Bash-it/bash-it/pull/1683/commits/833ff31810ce8d4521913844abd09798ef0911ec and https://github.com/Bash-it/bash-it/pull/1683/commits/4ae09b4a82cb76f023e292bc40c6993e4c5313f3